nextjs-app/app/layout.tsx

```diff
@@ -36,7 +35,6 @@ export default async function RootLayout({
           <main className="">{children}</main>
           <Footer />
         </section>
-        <SpeedInsights />
```

Comment:
Notice it will have connection issue on production build because of this, we can handle performance issue in the future. Take this off for now